usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/PetManagerClientTest.java

```diff
@@ -159,28 +211,113 @@ public void deletePet() throws ExecutionException, InterruptedException {
         given(taskManager.get()).willReturn(STATUS_OK);
         int responseCode = client.deletePet(ACCESS_TOKEN, USERNAME, petName);
         assertEquals(CODE_OK, expectedResponseCode,
-                responseCode);
+            responseCode);
     }
 
     @Test
-    public void updateField() throws ExecutionException, InterruptedException {
+    public void getSimpleFieldDouble() throws ExecutionException, InterruptedException {
+        given(taskManager.resetTaskManager()).willReturn(taskManager);
+        given(taskManager.execute(anyString(), anyString())).willReturn(taskManager);
+        given(taskManager.get()).willReturn(recKcalJson);
+        Object response = client.getSimpleField(ACCESS_TOKEN, USERNAME, petName, PetData.RECOMMENDED_KCAL);
+        assertEquals("Should return the gender value", RECOMMENDED_KCAL_EXAMPLE, response);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Should return the gender value" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5479466)